classroom.rb

```diff
@@ -0,0 +1,17 @@
+class Classroom
+  attr_accessor :label, :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  # classroom has many student
+  def many(student)
+    # saving each student object in an array
+    @student << student
+    # saving the current classroom object refered to as self
+    # to the student objects classroom variable
+    student.classroom = self
+  end
+end
```

Comment:
- [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md), '' `Create the belongs-to side (a student belongs to a classroom).` ", kindly update your `Student` class(in student.rb file not available for this PR) with a method that serves the `belongs-to` side the relationship between `Student and Classroom`. Something like so: 
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
#### Please this change is to be implemented in your student.rb file.